airflow/providers/elasticsearch/log/es_task_handler.py

```diff
@@ -321,7 +321,7 @@ def emit(self, record):
             setattr(record, self.offset_field, int(time() * (10**9)))
             self.handler.emit(record)
 
-    def set_context(self, ti: TaskInstance) -> None:
+    def set_context(self, ti: TaskInstance) -> None:  # type: ignore[override]
```

Comment:
Why is the ignore needed?

Reply:
Static checks fail otherwise saying it does not match the superclass's set_context method

Reply:
tried to describe earlier here https://github.com/apache/airflow/pull/32646#discussion_r1272979047

Reply:
<img width="1714" alt="Screenshot 2023-07-26 at 2 01 23 PM" src="https://github.com/apache/airflow/assets/10206082/2f3041e3-532a-4e59-96c9-64920f4f6bcb">